autoblocks/_impl/testing/run.py

```diff
@@ -300,8 +260,9 @@ def run_test_suite(
     # How many evaluators to run concurrently on the result of a test case
     max_evaluator_concurrency: int = 5,
 ):
-    init_global_vars_and_event_loop()
 
+    init_client()
```

Comment:
can you drop the extra space at line 263

Reply:
Dropped